features/aanschrijfwijze.feature

```diff
@@ -307,56 +333,56 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | B                       | J. barones Groenlo           | adellijke titel heeft bijbehorend hoffelijkheidstitel      |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | BS                      | J. Groenlo                   | adellijke titel heeft geen bijbehorend hoffelijkheidstitel |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | H                       | J. hertogin Groenlo          | adellijke titel heeft bijbehorend hoffelijkheidstitel      |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | HI                      | J. Groenlo                   | adellijke titel heeft geen bijbehorend hoffelijkheidstitel |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | G                       | J. gravin Groenlo            | adellijke titel heeft bijbehorend hoffelijkheidstitel      |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | GI                      | J. Groenlo                   | adellijke titel heeft geen bijbehorend hoffelijkheidstitel |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | M                       | J. markiezin Groenlo         | adellijke titel heeft bijbehorend hoffelijkheidstitel      |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | MI                      | J. Groenlo                   | adellijke titel heeft geen bijbehorend hoffelijkheidstitel |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | P                       | J. prinses Groenlo           | adellijke titel heeft bijbehorend hoffelijkheidstitel      |
+      | V        | Julia     | van         | Velzen        | P                      |                     | Groenlo               | PS                      | J. Groenlo                   | adellijke titel heeft geen bijbehorend hoffelijkheidstitel |
+      | V        | Julia     | van         | Velzen        | P                      |                     |                       | G                       | J. gravin van Velzen         | partner heeft geen geslachtsnaam                           |
```

Comment:
Goed testgeval! De titel is onlosmakelijk onderdeel van de achternaam. Dus wanneer geen geslachtsnaam wordt gebruikt, mag ook de hoffelijkheidstitel niet worden opgenomen. Nu suggereert de aanschrijfwijze dat de familie Van Velzen adellijk is.
Hier zorgt de rule van regel 49 dat bij het ontbreken van een partner naam een naamgebruik='E' moet worden gehanteerd. Dan zou de rule van regel 75 er vervolgens voor moeten zorgen dat de hoffelijkheidstitel niet mag worden gebruikt.